src/main/resources/languages/message_fr.properties

```diff
@@ -6325,8 +6325,8 @@ siteInfo.instruction.patient.nationality = S\u00E9lectionner "vrai" pour \
                                            nationalit\u00E9 des patients soit \
                                            entr\u00E9 dans la base de \
                                            donn\u00E9es.
-siteInfo.lab.director                    = Le nom du directeur du laboratoire \
-                                           sur les rapports
+siteInfo.lab.director                    = Nom du responsable de laboratoire \ 
```

Comment:
multi-line messages in these files need to end with a '\'. Putting a space after the '\' will cause it to ignore the second line.

Reply:
@CalebSLane kindly review